src/window-service.js

```diff
@@ -268,7 +268,8 @@
                     resizable: false,
                     minHeight: 500,
                     minWidth: 230,
-                    maximizable: false
+                    maximizable: false,
+                    shadow: true
```

Comment:
:( Sorry my fault here, wasn't clear in my previous comment. I don't think you need this one either. Given that it might be worth a squash too...